preprocess.yaml

```diff
@@ -39,68 +42,77 @@ targets:
 
-  cache/pre_state_boundaries_zoom.json:
+      
+# --- shift back scales after simplication of counties --- #
+  cache/pre_county_boundaries_USA.json:
+    command: rescale_write_shps(
+      target_name, "cache/simpled_county_boundaries_USA.json", STATEFP = I(c('72','78','02')), scale = I(c(1/3, 1/3, 2)), quantize=I(1e6))
+
+  cache/pre_county_boundaries_mobile.json:
+    command: rescale_write_shps(
+      target_name, "cache/simpled_county_boundaries_mobile.json", STATEFP = I(c('72','78','02')), scale = I(c(1/3, 1/3, 2)), coerce_dropped = I(TRUE), quantize=I(1e7)) 
+
+  cache/pre_county_boundaries_zoom.json:
+    command: rescale_write_shps(
+      target_name, "cache/simpled_county_boundaries_zoom.json", STATEFP = I(c('72','78','02')), scale = I(c(1/3, 1/3, 2)), coerce_dropped = I(TRUE), quantize=I(1e8)) 
+
+# --- use topojson versions of counties to create state boundaries --- #
+
+  cache/pre_state_boundaries_USA.json:
     depends: scripts/preprocess/custom_projection.js
     command: process_counties_to_states(
-      outfile=target_name, objects=I('states'), quantize=I(1e8), zoom_dims=I(TRUE),
-      county_topojson='cache/pre_county_boundaries_zoom.json',
+      outfile=target_name, objects=I('states'), quantize=I(1e6), zoom_dims=I(FALSE),
+      county_topojson='target/data/county_boundaries_USA.json',
       state_dict_file='cache/pre_state_dictionary.json', script_file = 'scripts/preprocess/process_counties_to_states.js') 
-  
-  cache/pre_all_boundaries_zoom.json:
+
+  cache/pre_state_boundaries_mobile.json:
     depends: scripts/preprocess/custom_projection.js
     command: process_counties_to_states(
-      outfile=target_name, objects=I('states-and-counties'), quantize=I(1e8), zoom_dims=I(FALSE),
-      county_topojson='cache/pre_county_boundaries_zoom.json',
-      state_dict_file='cache/pre_state_dictionary.json', script_file = 'scripts/preprocess/process_counties_to_states.js') 
-  
-  # simplify somewhere between USA and zoom top optimize for mobile
-  cache/pre_county_boundaries_mobile.json:
-    command: process_counties(
-      outfile=target_name, county_sp=county_boundaries_census_sp,
-      simplify=I(1e-7), quantize=I(1e7),
-      county_dict_file='cache/pre_county_dictionary.json', script_file='scripts/preprocess/process_counties.sh')
-  
-  cache/pre_state_boundaries_mobile.json:
+      outfile=target_name, objects=I('states'), quantize=I(5e6), zoom_dims=I(TRUE),
+      county_topojson='target/data/county_boundaries_mobile.json',
+      state_dict_file='cache/pre_state_dictionary.json', script_file = 'scripts/preprocess/process_counties_to_states.js')   
+
+  cache/pre_state_boundaries_zoom.json:
     depends: scripts/preprocess/custom_projection.js
     command: process_counties_to_states(
       outfile=target_name, objects=I('states'), quantize=I(1e8), zoom_dims=I(TRUE),
-      county_topojson='cache/pre_county_boundaries_mobile.json',
+      county_topojson='target/data/county_boundaries_zoom.json',
```

Comment:
this looks like an unstable output file location...did you intend to keep it this way?

Reply:
this was a relic from editing this to push to target for local testing. will fix